README.md

```
@@ -31,6 +31,17 @@ Hate writing docs? Let your tests do it. Your API docs stay in sync—no surpris
 [Redoc-style]: https://redocly.github.io/redoc/
 [HTML output]: http://redocly.com/demo/openapi/museum-api/operations/getmuseumhours
 
+## Playground
+
+You can try itdoc right away in your browser — no need to install Node.js or set up a local project.
+Visit https://itdoc.kr/playground to experiment with itdoc and see how API documentation is
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)